### PR TITLE
Style and other improvements for get_started and ros_installation

### PR DESCRIPTION
**Pull request by j-rivero**
One fix or improvement per commit:

- Use warning div for ROS note
- Add Windows to the table of distros
- Fix old gazebo classic reference
- Use plain bold style
- Improve header description
- Clarify URL for osrfoundation repo

### CODE REVIEW

File: get_started.md [hunk 1]
```diff
@@ -21,9 +23,7 @@ packages available for the platform to use:
 | Mac Monterey | [Gazebo Garden](/docs/garden/install_osx) (recommended), [Gazebo Fortress](/docs/fortress/install_osx) and [Gazebo Citadel](/docs/citadel/install_osx)
 | Mac BigSur | [Gazebo Garden](/docs/garden/install_osx) (recommended), [Gazebo Fortress](/docs/fortress/install_osx) and [Gazebo Citadel](/docs/citadel/install_osx)
 | Mac Catalina | [Gazebo Garden](/docs/garden/install_osx) (recommended), [Gazebo Fortress](/docs/fortress/install_osx) and [Gazebo Citadel](/docs/citadel/install_osx)
-
-Windows support via conda-forge is not fully functional, there are known runtime problems
-[Gazebo for Windows Issue](https://github.com/gazebosim/gz-sim/issues/168).
+| Windows | Support via conda-forge is not fully functional, there are known runtime problems [Gazebo for Windows Issue](https://github.com/gazebosim/gz-sim/issues/168).
```
Comment (kscottz) [inline get_started.md:26]:
```suggestion
| Windows | Support via Conda-Forge is not fully functional, as there are known runtime issues [see this issue for details](https://github.com/gazebosim/gz-sim/issues/168).
```

File: ros_installation.md [hunk 2]
```diff
@@ -42,21 +42,21 @@ The easiest way of installing Gazebo is to use binary packages. There are two ma
...
- * ***packages.osrfoundation.org***
+ * **packages.osrfoundation.org**
    * Gazebo Citadel
    * Gazebo Edifice
    * Gazebo Fortress
    * Gazebo Garden
 
 This means that including the osrfoundation repository is not strictly needed
-to get the Gazebo binary package in Ubuntu. It can be installed from the ROS
+to get the Gazebo binary packages it can be installed from the ROS
```
Comment (kscottz) [inline ros_installation.md:59]:
```suggestion
to get the Gazebo binary packages, it can be installed from the ROS
```

**Review by kscottz — CHANGES_REQUESTED** [review]
Made a few suggestions. Toss those in and this can be merged.

**Comment by j-rivero** [comment]
> Made a few suggestions. Toss those in and this can be merged.

Thanks Kat!

**Pull request merged**